examples/cms-buttercms/.gitignore

```diff
@@ -32,3 +32,6 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# other
+.vscode
```

Comment:
The test was failing because of this - all examples need to have the same .gitignore 👍

Reply:
I've committed this change